README.md

```diff
@@ -53,3 +53,7 @@ If you build your module with this command:
 You could generate `compile_commands.json` with:
 
     $ python .vscode/generate_compdb.py -O $KDIR $PWD
+
+Ensure that your module has been built and contains some `*.o.cmd` files before generating `compile_commands.json`.
+
+`$KDIR` must be the directory of your kernel source, not your kernel build.
```

Comment:
Hm. Your "steps to reproduce" in https://github.com/amezin/vscode-linux-kernel/issues/9 used build dir as `KDIR` and, if I remember correctly, it worked

Reply:
Yes, the script built `compile_commands.json`, but the file it generated was incorrect. VSCode was able to find some headers but not all of them. Switching the directory to the kernel source fixed the issue. However, I'm now unable to reproduce this behavior, so I think there was something wrong with my setup.

Reply:
Re-tested just now, and, actually, VSCode can't find headers (i. e. `compile_commands.json` is incorrect) if you pass `-O kernel-sources` to `generate_compdb.py`. It must be `-O build-dir` (and `make -C kernel-source M=$PWD modules` fails too).

If you test things like this, you should restart VSCode when you change something - they claim to support live reloading of `compile_commands.json`, but, from my experience, it's not very robust.

Reply:
I meant that I did an in-tree kernel build, then pointed `-O` to the kernel source+build dir. So I think the correct value of `$KDIR` is pretty clearly explained by the README in its current form.